test/support/feature_helper.rb

```diff
@@ -0,0 +1,28 @@
+module FeatureHelper
```

Comment:
This is a pretty nice abstraction. It reminds me a lot of Cucumber testing. If this was your first thought with handling tests I think you should really look into it. Here is a quick link from their github. 
https://cucumber.io/blog/2017/05/15/intro-to-bdd-and-tdd